clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -788,36 +788,27 @@ protected synchronized void markCoordinatorUnknown(boolean isDisconnected) {
     }
 
     /**
-     * Get the current generation state if the group is stable.
-     * @return the current generation or null if the group is unjoined/rebalancing
+     * Get the current generation state
+     *
+     * @return the current generation
```

Comment:
Can you maybe extend the docs a bit to explain that this always returns the latest stable generation, regardless of whether it is currently stable? Is that an accurate description?